readme.md

```diff
@@ -1,6 +1,6 @@
 # Zafira Orga telegram bot
 
-Henrik sein bot!
+Henrik sein bot! Wie geilasdfasdf
```

Comment:
Was is dat denn. Scheint mir etwas dumm zu sein.

Reply:
Oh, stimmt. Das fixe ich!.